tests/cpp/test_reduction_pointwise.cpp

```diff
@@ -158,4 +158,39 @@ TEST_F(NVFuserTest, InnerReductionUnrollVectorization) {
   testValidate(&fusion_copy, cg_outputs, {t0}, __LINE__, __FILE__);
 }
 
+// https://github.com/NVIDIA/Fuser/issues/3811
+TEST_F(NVFuserTest, ReductionSchedulerWithAdditionalID) {
+  auto fusion_ptr = std::make_unique<Fusion>();
+  auto& fusion = *fusion_ptr;
+  FusionGuard fg(fusion_ptr.get());
+
+  // tv0 [ b0, i1 ]
+  auto tv0 = makeContigConcreteTensor({1, -1});
+  fusion.addInput(tv0);
+  // tv1 [ i2, i1 ]
+  // since i2 is not covered by reference TV, reduction scheduler should reject
```

Comment:
nit: suggest revising comments to include "current scheduler picks tv0 as the reference tv, it is transformed and propagated to other tvs"